src/state/actions/dispatchArNSUpdate.ts

```diff
@@ -67,7 +67,10 @@ export function dispatchArNSUpdate({
           processId: id,
         })
           .getHandlers()
-          .catch(console.error);
+          .catch((e) => {
+            console.error(e);
+            return null;
```

Comment:
This was a noisy error from react-query because it needs a serializable return (null is, undefined isnt)